apps/frontend/src/app/poll/choose-events/choose-events.component.ts

```diff
@@ -202,13 +210,32 @@ export class ChooseEventsComponent implements OnInit {
     } else if (maxParticipants && this.participants.length >= maxParticipants) {
       this.closedReason = 'This poll has reached it\'s maximum number of participants. You can no longer submit your vote.';
       this.showResults = true;
-    } else {
-      this.closedReason = undefined;
-      this.showResults = !this.poll?.settings?.blindParticipation || this.isAdmin || this.userVoted();
     }
   }
 
-  private clearSelection(){
+  private updateHiddenReason() {
+    this.showResults = this.isAdmin || this.userVoted() ||
```

Comment:
This line only has an effect in the AFTER_PARTICIPATING branch of the switch, in the other cases it is overriden to true.

Reply:
Fixed by 652196a.